test/tests/deformation/mesh-velocity-areas/nek.i

```diff
@@ -0,0 +1,64 @@
+[Mesh]
+  type = NekRSMesh
+  order = SECOND
+  boundary = '2'
+  parallel_type = replicated
+  displacements = 'disp_x disp_y disp_z'
+  moving_mesh = true
+[]
+
+[Problem]
+  type = NekRSProblem
+  casename = 'pipe'
+  synchronization_interval = parent_app
+[]
+
+[AuxVariables]
+  [dummy]
+  []
+[]
+
+# This AuxVariable and AuxKernel is only here to get the postprocessors
+# to evaluate correctly. This can be deleted after MOOSE issue #17534 is fixed.
+[AuxKernels]
+  [dummy]
+    type = ConstantAux
+    variable = dummy
+    value = 0.0
+  []
+[]
+
+[Executioner]
+  type = Transient
+  [TimeStepper]
+    type = NekTimeStepper
+  []
+  [Quadrature]
+    type = GAUSS_LOBATTO
+    order = SECOND
+  []
+[]
+
+[Postprocessors]
+  [nekbdry_icar]
+    type = NekSideIntegral
+    field = unity
+    boundary = '2'
+    use_displaced_mesh = true
+    execute_on = INITIAL
+  []
+  [nekbdry_ar]
+    type = NekSideIntegral
+    field = unity
+    boundary = '2'
+    use_displaced_mesh = true
```

Comment:
Do you think we should perhaps set `use_displaced_mesh` automatically, based on the moving mesh settings? I'm sort of on the fence - it would be convenient for the user, but then the user would need to remember to do `use_displaced_mesh = true` for any postprocessors that we're not defining in Cardinal (i.e. any postprocessor in the MOOSE framework itself).